src/Uno.UWPSyncGenerator/DocGenerator.cs

```diff
@@ -140,6 +140,18 @@ void BuildMemberLists()
 									_sb.AppendParagraph($"In addition, {formattedViewName} has Uno-specific documentation {Hyperlink("here", customDocLink)}.");
 								}
 
+								var galleryLink = GetGalleryLink(viewName);
+								if (galleryLink != null)
+								{
+									_sb.AppendParagraph($"To better understand how {formattedViewName} works, you can use the Gallery {Hyperlink("here", galleryLink)}.");
+								}
+
+								var playgroundLink = GetPlaygroundLink(viewName);
+								if (playgroundLink != null)
+								{
+									_sb.AppendParagraph($"To better understand how {formattedViewName} works, you can use the Gallery {Hyperlink("here", playgroundLink)}.");
```

Comment:
This is working better now: https://unodocsprstaging.z13.web.core.windows.net/pr-14833/articles/implemented/microsoft-ui-xaml-controls-combobox.html, We need to adjust the text now. It's probably best to merge the two to end up like this: `To better understand how XX works, you can use the [Uno Gallery] or the [Uno Playground]`. A string join with ` or ` should do the trick.

Reply:
I was already forwarding an update to look like this.

	_To better understand how XX works, you can use the Gallery here.
	Or run your own tests on XX here on Playground._

Would it look better in one line?

Had I thought about passing it on to Ludvig to think of a way to make the change more prominent?
Or suddenly create a topic for it.
It can help users a lot to be able to look at the rules (microsoft link) and easily see how it works in the gallery and test it in the playground.

Do you think it's worth it or is that ok?

Reply:
![image](https://github.com/unoplatform/uno/assets/116665025/c9e99387-7468-4bb2-8964-274ad84db117)

Reply:
@jeromelaban 

The way the image/link bellow looks is it good or should I change it to just one line?

![image](https://github.com/unoplatform/uno/assets/116665025/d1f976d5-2683-4d1c-8c1b-2876d0d4160a)

e.g https://unodocsprstaging.z13.web.core.windows.net/pr-14833/articles/implemented/microsoft-ui-xaml-controls-combobox.html